src/cascadia/TerminalSettingsModel/Command.cpp

```diff
@@ -33,19 +33,38 @@ static constexpr std::string_view SchemeNameToken{ "${scheme.name}" };
 
 namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 {
-    Command::Command()
+    Command::Command() :
+        _subcommands{ winrt::single_threaded_map<winrt::hstring, Model::Command>() }

```

Comment:
I'm just as against this now as I was when @zadjii-msft used it: it allocates a whole map for each command, the vast majority of which will be completely unused.